packages/devtools_shared/lib/src/devtools_api.dart

```diff
@@ -64,3 +64,12 @@ const String apiGetTestAppSizeFile = '${apiPrefix}getTestAppSizeFile';
 const String baseAppSizeFilePropertyName = 'appSizeBase';
 
 const String testAppSizeFilePropertyName = 'appSizeTest';
+
+/// Serves any available extensions and returns a list of their configurations
+/// to DevTools.
+const String apiServeAvailableExtensions =
+    '${apiPrefix}serveAvailableExtensions';
+
+const extensionRootPathPropertyName = 'rootPath';
```

Comment:
Nit: documentation for these two property strings. Maybe add types to be consistent with the rest of the constants here?